tokens/src/themes/light/components/Alert.json

```diff
@@ -21,6 +21,16 @@
         "info": { "source": "$alert-info-border-color", "$value": "{color.theme.border.info}" },
         "danger": { "source": "$alert-danger-border-color", "$value": "{color.theme.border.danger}" },
         "warning": { "source": "$alert-warning-border-color", "$value": "{color.theme.border.warning}" }
+      },
+      "actions": {
+        "overrides": {
+          "variants": {},
+          "examples": {
+            "variants": {
+              "primary": "brand"
+            }
+          }
```

Comment:
[inform] This `examples` key serves as a way to provide some inline-documentation in the JSON tokens themselves around usage for the `actions.overrides.variants`.